.github/workflows/ci.yaml

```diff
@@ -4,6 +4,10 @@ on:
       - master
   pull_request:
 name: ci
+env:
+  CI: true
```

Comment:
I will admit I don't know what `CI` does :) At a glance, it looks like this is set for you by default with actions?
https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Reply:
It's a common CI variable :) I have to admit I didn't check since a long time if they specify it, but in the beginning they definitely didn't. But hey, it's been 1 and a half year since they launched actions.

I can remove CI then.